airflow/providers/yandex/hooks/yandex.py

```diff
@@ -146,9 +146,18 @@ def _get_credentials(self) -> dict[str, Any]:
             return {'token': oauth_token}
 
     def _get_field(self, field_name: str, default: Any = None) -> Any:
-        """Fetches a field from extras, and returns it."""
-        long_f = f'extra__yandexcloud__{field_name}'
-        if hasattr(self, 'extras') and long_f in self.extras:
-            return self.extras[long_f]
-        else:
+        """Get field from extra, first checking short name, then for backcompat we check for prefixed name."""
+        if not hasattr(self, 'extras'):
             return default
+        backcompat_prefix = 'extra__yandexcloud__'
+        if field_name.startswith('extra_'):
```

Comment:
Non-blocking suggestion, is there a reason to not use startswith(backcompat_prefix)?

Reply:
It actually looks like you id this in some of the newer ones.